app/views/commontator/comments/_body.html.erb

```diff
@@ -8,10 +8,11 @@
 <!--- Show timestamps in comments that have a linked annotation --->
 <% annotation = comment.annotation %>
 <% unless annotation.nil? %>
-  <% medium = annotation.medium %>
+  <% medium_id = annotation.medium.id %>
```

Comment:
Remove the `id` stuff, see above.

Reply:
But this will send the full medium information to the frontend, not just the id, won't it? If that's the case, I'd rather go for `.id` to reduce bandwidth (although that might be neglegible to be honest)

Reply:
What I said is probably premature optimization and Rails can probably handle that case just fine and still quite performant, so I just applied your suggestion.